essays/AITBW-roadToDune/index.md

```diff
@@ -0,0 +1,83 @@
+# Dr. Strangelove or: How I handled my stubbornness to become a web developer
+
+Becoming a developer can be a pretty underwhelming task these days, especially if you delve into the [JavaScript](https://hackernoon.com/how-it-feels-to-learn-javascript-in-2016-d3a717dd577f) land, where _the new kid on the block_ can be thrown into obscurity in a matter of days.
+
+I faced a completely different situation back when I began college in 2010, which is the story I'd like to tell you today.
+
+I enrolled in college around mid/late 2010 under an "Information Engineer" undergrad, which is mostly focused in the importance/value of data and how we can take advantage of IT systems to make the most out of it.
+
+When I began college, one of my biggest weaknesses from high school were maths. My education had a very high standard but I just couldn't make sense out of them —as a matter of fact: I struggled with maths during **all** of my high school education.
+
+You must be asking: Then why did you went for Engineering?
+
+1. I love a good challenge
+2. I didn't see myself in the future as anything other than an Engineer
+
+So there I went.
+
+My first real programming course was "Programación I" or "Programming 101", which was taught using C#.
+
+> I failed.
```

Comment:
The markdown syntax for `>` is used to quote something. Please remove it, since this isn't a quote.

Reply:
I'm using the `quote` syntax to emphasize something, similar to this

![screenshot-2018-2-18 the y combinator no not that one ayaka nonaka medium](https://user-images.githubusercontent.com/9470839/36359692-f1d86e98-14f3-11e8-90a6-7deefd4b068e.png)

Reply:
I wouldn’t use it as part of the same voice you use it for the rest of the text. Quotations should be used to denote content that is coming from another source, to denote them to be as they were when observed. This reminds me of these cases where people use quotation marks on the air to put emphasis or even to denote irony.

Reply:
For highlighting we have bold and italic, that should be sufficient.